packages/devtools_app/lib/src/shared/memory/adapted_heap_data.dart

```diff
@@ -56,6 +56,8 @@ class HeapObjectSelection {
   }
 }
 
+typedef HeapDataObtainer = AdaptedHeapData Function();
```

Comment:
nit: rename to HeapDataCallback? "Obtainer" makes it sound like this is a Dart Class and not a function typedef